configs/statamic_3.json

```diff
@@ -1,33 +1,42 @@
 {
   "index_name": "statamic_3",
   "start_urls": [
+    "https://statamic.dev",
```

Comment:
This line should be at the end of the `start_urls` array to validate the next conditions